kv/store_base.go

```diff
@@ -26,7 +26,7 @@ func EncIDKey(ent Entity) ([]byte, string, error) {
 	return id, "ID", err
 }
 
-func EncOrgNameKeyFn(caseSensitive bool) func(ent Entity) ([]byte, string, error) {
+func EncOrgNameKey(caseSensitive bool) func(ent Entity) ([]byte, string, error) {
```

Comment:
add comments for exported func

Reply:
I agree, I'm strugglign with naming convention here. The `EncOrgNameKey` key is a key of the org and name.